src/tools/PlaywrightSharp.Tooling/DriverDownloader.cs

```diff
@@ -90,12 +91,7 @@ private static Task<string> GetUpstreamReadmeAsync(string playwrightVersion)
         private async Task DownloadDriverAsync(DirectoryInfo destinationDirectory, string driverVersion, string platform, string runtime)
         {
             Console.WriteLine("Downloading driver for " + platform);
-            string cdn = "https://playwright.azureedge.net/builds/driver";
-
-            if (driverVersion.Contains("next"))
-            {
-                cdn += "/next";
-            }
+            string cdn = "https://playwright.azureedge.net/builds/driver/next";
```

Comment:
Is this right, for a non-next version of `<DriverVersion>1.9.0-1614037901000</DriverVersion>`?

Reply:
I took that from this change https://github.com/microsoft/playwright-python/pull/522/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L59

Reply:
Hm, @aslushnikov can you clarify? Are we now supposed to use `/next` for all driver downloads?

Reply:
@kblok Andrey just confirmed this is right, so we're fine!